x/compute/internal/keeper/query_plugins.go

```diff
@@ -92,9 +98,40 @@ func (e QueryPlugins) Merge(o *QueryPlugins) QueryPlugins {
 	if o.Mint != nil {
 		e.Mint = o.Mint
 	}
+	if o.Gov != nil {
+		e.Gov = o.Gov
+	}
 	return e
 }
 
+func GovQuerier(keeper *gov.Keeper) func(ctx sdk.Context, request *wasmTypes.GovQuery) ([]byte, error) {
+	return func(ctx sdk.Context, request *wasmTypes.GovQuery) ([]byte, error) {
+		if request.Proposals != nil {
+			proposals := keeper.GetProposals(ctx)
+
+			if len(proposals) == 0 {
+				return json.Marshal(wasmTypes.ProposalsResponse{
+					Proposals: []wasmTypes.Proposal{},
+				})
+			}
```

Comment:
I think you don't need this block. If `proposals` is empty, then the for loop will just not do anything and continue.

Reply:
I do. Otherwise it returns `{}`